### PR TITLE
pioneering-rlj:0.6.0

### PR DESCRIPTION
**Pull request by daskol**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Template for Reinforcemnt Learning Conference/Journal (RLC/RLJ) (see details in upstream https://github.com/daskol/typst-templates/issues/27 and https://github.com/daskol/typst-templates/issues/28).

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE
- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/pioneering-rlj/0.6.0/thumbnail.png
Comment (elegaanz) [inline packages/preview/pioneering-rlj/0.6.0/thumbnail.png:1]: Your thumbnail has a transparent background, which will not look so nice in the web app. I would recommend regenerating it with a white one.

File: packages/preview/pioneering-rlj/0.6.0/rlj.typ [hunk 1]
```diff
@@ -0,0 +1,631 @@
+/**
+ * rlj.typ
+ *
+ * Reinforcement Learning Journal/Conference (RLJ/RLC) template.
+ */
+
+#let std-bibliography = bibliography  // Due to argument shadowing.
```
Comment (elegaanz) [inline packages/preview/pioneering-rlj/0.6.0/rlj.typ:7]: For info, you can use `std.bibliography` to disambiguate since Typst 0.12.0.

**Review by elegaanz — COMMENTED** [review]
Thank you for the package :) Just a small thing you may want to fix about your thumbnail, and it should be good to merge.

**Comment by daskol** [comment]
Thank you for your comments. Fixed.

**Comment by elegaanz** [comment]
Thanks!

**Pull request merged**